verification/curator-service/ui/src/components/AutomatedBackfill.tsx

```diff
@@ -86,6 +114,36 @@ export default function AutomatedBackfill(props: Props): JSX.Element {
     const classes = useStyles();
     const [errorMessage, setErrorMessage] = React.useState('');
     const [successMessage, setSuccessMessage] = React.useState('');
+    const [sourceId, setSourceId] = React.useState('');
+    const [uploadId, setUploadId] = React.useState('');
+    const [uploadStatus, setUploadStatus] = React.useState('');
+
+    useInterval(async () => {
+        if (uploadStatus === 'IN_PROGRESS') {
+            const response = await axios.get<SourceResponse>(
+                `/api/sources/${sourceId}`,
```

Comment:
at some point we should just have a get endpoint for /api/sources/$sourceId/uploads/$uploadId

Reply:
+1. Like they say, you-only-list-once.